core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -346,16 +352,20 @@ class MapUnit {
         return unit
     }
 
-    fun canUpgrade(): Boolean {
+    /** @param preemptively: Ignore possible tech/culture requirements. Used for upgrading units
```

Comment:
Isn't 'to preempt' something like 'to react to an event before it even happens', Like preemptively nuking Antarctica because the Nazis could be hiding there? Or preemptively banning @xlenstra from this repo because they _might_ deliver bullshit in some future PR :innocent: ?

Maybe 'unconditionally'? I mean preempt away or even 'xyzzy' would be OK as the Kdoc is fine, but...

Reply:
I used 'preemptively' as 'upgrade to a unit before we know that unit / are allowed to upgrade', but I suppose it doesn't quite match the definition of 'preemptively'. I'll see if I can find something better